### PR TITLE
feat: add webui backend to rock

### PR DESCRIPTION
**Pull request by patriciareinoso**
# Description

The Rock container image for the NMS should contain both the Next.JS frontend (NMS) and backend components (WebUI). 

This PR:
- Build the frontend
- Copy the static files over to the `webconsole/ui/frontend_files` directory
- Build the backend with the ui go build tag.

**Remove integration tests:** webui is the service that runs on the rock. We cannot test that we have access to the UI as before because, in order to get the webui up and running it needs to connect to mongoDB. If the connection fails, an exception is raised.

This PR depends on:
- https://github.com/omec-project/webconsole/pull/191
- https://github.com/canonical/sdcore-nms/pull/619
- https://github.com/omec-project/webconsole/pull/194

# Checklist:

- [ ] My code follows the [style guidelines](/CONTRIBUTING.md) of this project
- [ ] I have performed a self-review of my own code.
- [ ] I have made corresponding changes to the documentation.
- [ ] I have added tests that validate the behaviour of the software.
- [ ] I validated that new and existing unit tests pass locally with my changes.
- [ ] Any dependent changes have been merged and published in downstream modules.

### CODE REVIEW

File: rockcraft.yaml [hunk 1]
```diff
@@ -9,35 +9,56 @@ platforms:
   amd64:
 
 services:
```
Comment (patriciareinoso) [inline rockcraft.yaml:11]: can I remove the service just like the other rocks?

File: rockcraft.yaml [hunk 2]
```diff
@@ -9,35 +9,56 @@ platforms:
...
+      - app
+    prime:
+      - -app
+  webui:
+    after:
+      - nms
+    plugin: go
+    source: https://github.com/patriciareinoso/webconsole.git
+    source-type: git
+    source-commit: 93f675baa7eb5f5df37e4013cf4cda0beeba5d84
```
Comment (patriciareinoso) [inline rockcraft.yaml:48]: replace commit by tag

File: rockcraft.yaml [hunk 3]
```diff
@@ -9,35 +9,56 @@ platforms:
...
+      SWAGGER_HOST: 10.1.182.9
 
 parts:
-
   nms:
     plugin: nil
-    source: .
+    source: https://github.com/canonical/sdcore-nms.git
+    source-type: git
+    source-commit: 5dd01d6c484331d4134cd0d11c60358f4cb160f3
```
Comment (patriciareinoso) [inline rockcraft.yaml:26]: replace commit by 
`source: . ` 
as before

File: rockcraft.yaml [hunk 4]
```diff
@@ -9,35 +9,56 @@ platforms:
   amd64:
```
Comment (patriciareinoso) [inline rockcraft.yaml:9]: What version to use?

File: rockcraft.yaml [hunk 5]
```diff
@@ -9,35 +9,56 @@ platforms:
...
 
 services:
-  nms:
-    command: npm run start
+  webui:
```
Comment (patriciareinoso) [inline rockcraft.yaml:12]:
```suggestion
  nms:
```

File: rockcraft.yaml [hunk 6]
```diff
@@ -27,14 +17,36 @@ parts:
...
+      cp -r out/* ${CRAFT_PART_INSTALL}/app/
+    stage:
+      - app
+    prime:
+      - -app
+  webui:
+    after:
+      - nms
+    plugin: go
+    source: https://github.com/patriciareinoso/webconsole.git
```
Comment (gruyaume) [inline rockcraft.yaml:34]: We should probably wait for this change to be tagged before merging. Gab should be able to create the new tag.

File: .github/workflows/integration_tests.yaml
Comment (gruyaume) [inline .github/workflows/integration_tests.yaml:1]: Why get rid of the integration tests?

**Review by gruyaume — CHANGES_REQUESTED** [review]

**Review by gruyaume — APPROVED** [review]

**Pull request merged**